### ISSUE TITLE
"Could not find matching DAC for this runtime" for Linux core dump

### ISSUE THREAD

**Issue by loop-evgeny** [microsoft/clrmd#210]
ClrMd 1.0.2, .NET Core 2.1.6 on Ubuntu 18.04.1 (x64).

I created a dump using the .NET Core `createdump` utility. I can open it using `DataTarget.LoadCoreDump` but then `CreateRuntime` fails:

```
Unhandled Exception: System.IO.FileNotFoundException: Could not find matching DAC for this runtime.
   at Microsoft.Diagnostics.Runtime.DataTarget.CreateRuntime(ClrInfo clrInfo)
```

I'm running this on the same machine where the dump was created, so mismatching .NET Core versions shouldn't be an issue.

**Comment by leculver** [comment]
Thanks for the report.  For now, you can specify the path directly with ClrInfo.CreateRuntime(path, validate=false).  This will be fixed in ClrMD 1.1 (or before).

**Comment by leculver** [comment]
This was fixed a while back.  If running on the same machine that took the coredump, `CreateRuntime()` will find `libmscordaccore.so` on disk.  If not, you will have to locate the dac and provide it as a parameter to CreateRuntime.  Unfortunately ClrMD does not know how to request a linux dac from the symbol server.  I'm not planning to fix that until ClrMD 2.0.
